pkg/kritis/review/review_test.go

```diff
@@ -222,13 +222,15 @@ func TestReviewGAP(t *testing.T) {
 			hasRequiredAtts: false,
 			shouldErr:       false,
 		},
+		// TODO: After payload check implemented in cryptolib,
+		// `hasRequiredAtts` should be false, and `shouldErr` should be true.
```

Comment:
What is this trying to test right now? I'd rather disable a test that does not work because something is unimpliemented than have a test that expects incorrect behaviour

Reply:
Reverted to original behavior.